main.go

```diff
@@ -30,13 +30,13 @@ import (
 )
 
 func main() {
-	username, password, skipSslValidation, reap, recursive, apiUrl, serviceName, expiryInterval := arg.Parse(os.Args, os.Stdout, os.Exit)
+	username, password, skipSslValidation, reap, recursive, apiUrl, serviceName, planName, expiryInterval := arg.Parse(os.Args, os.Stdout, os.Exit)
 
 	if !reap {
 		fmt.Printf("DRY RUN ONLY!\n")
 	}
 
-	fmt.Printf("Reaping instances of service %s older than %s in %s as %s...\n", serviceName, durafmt.Parse(expiryInterval), apiUrl, username)
+	fmt.Printf("Reaping instances of service %s with plans %s older than %s in %s as %s...\n", serviceName, planName, durafmt.Parse(expiryInterval), apiUrl, username)
```

Comment:
I know what you mean here but it reads a bit like we're reaping plans which were defined before a certain date or something like that.

Maybe something like `Reaping instances of 'p-my-service / free' older than...` or `Reaping instances of the 'free' plan of 'p-my-service' older than...`?

Reply:
`Reaping instances of the 'service-plan-name-0' plan of 'service-name' older than 10 hours in https://127.0.0.1:63427 as username...`